package.json

```diff
@@ -65,11 +65,11 @@
     "build:traditional": "rollup -c --config-traditional",
     "postbuild": "pnpm patch-d-ts && pnpm copy && pnpm patch-old-ts && pnpm patch-esm-ts",
     "prettier": "prettier \"*.{js,json,md}\" \"{examples,src,tests,docs}/**/*.{js,jsx,ts,tsx,md,mdx}\" --write",
-    "eslint": "eslint --no-eslintrc --c .eslintrc.json --fix '*.{js,ts}' '{src,tests}/**/*.{ts,tsx}'",
+    "eslint": "eslint '*.{js,mjs,ts}' '{src,tests}/**/*.{ts,tsx}' --fix ",
     "test": "pnpm run '/^test:.*/'",
     "test:format": "prettier '*.{js,json,md}' '{examples,src,tests,docs}/**/*.{js,jsx,ts,tsx,md,mdx}' --list-different",
     "test:types": "tsc --noEmit",
-    "test:lint": "eslint  --no-eslintrc --c .eslintrc.json '*.{js,ts}' '{src,tests}/**/*.{ts,tsx}'",
+    "test:lint": "eslint '*.{js,mjs,ts}' '{src,tests}/**/*.{ts,tsx}'",
```

Comment:
Add `mjs` because we have `eslint.config.mjs` file

Reply:
I would like `"eslint ."` here too, and also for `--fix` above.

Reply:
@dai-shi The reason I configured it this way is simply to download `./pnpm-lock.yaml` and run ESLint. Since the dependencies `@react-three/drei`, `@react-three/fiber`, and `@react-three/postprocessing`, etc... are not included within it, errors occur.

However, I understand your intention, so after this PR is merged, I will create a new PR to address it accordingly.

Reply:
Ohhh, now I remember that. Sorry about the repeated asking.
Hm, I don't know honestly what solution I like. Definitely we don't want monorepo. So, the current solution seems good.

I would probably try adding `examples/` in `ignores` in eslint.config.mjs. How you tried before?

Reply:
@dai-shi 

> I would probably try adding `examples/` in `ignores` in eslint.config.mjs. How you tried before?

Oh, I hadn’t thought of that approach, that sounds great, i gonna reflect your idea 👍

Reply:
@dai-shi 

> Hm, I don't know honestly what solution I like. Definitely we don't want monorepo. So, the current solution seems good.

After it is merged, I will think of a better way to utilize `eslint` for the `examples`.